.travis.yml

```diff
@@ -62,9 +61,12 @@ matrix:
         - mysql
         - postgresql
   allow_failures:
-  - dist: bionic
-    python: 3.9-dev
-    env:
+    - arch: arm64
```

Comment:
I think you have have to have it in *both* sections (allow_failures) and the top section; they must be identical.

Reply:
Thanks for the suggestion. I have added it as suggested and all checks are passing as well. Please review and let me know if any more updates are required.